app/Resources/views/themes/tastefulplanet/pagelayout.html.twig

```diff
@@ -14,18 +14,17 @@
         <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700" rel="stylesheet">
         <link href="https://use.fontawesome.com/releases/v5.0.1/css/all.css" rel="stylesheet">
 
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
-        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
-
         {% block page_head %}
             <title>{{ title|default('Home') }}</title>
         {% endblock %}
 
         {% block head_styles %}
-            {% stylesheets filter='cssrewrite' '@css_default' %}
-                <link rel="stylesheet" type="text/css" href="{{ asset_url }}">
+            {% stylesheets filter='scssphp'
+                'bundles/ezplatformadminui/scss/ezplatform-bootstrap.scss'
+                'bundles/ezplatformadminui/scss/ezplatform.scss'
```

Comment:
I don't think you need `ezplatform.scss`. Those are styles for admin-ui.

Reply:
fixed